pkg/plugins/platform/utils.go

```diff
@@ -145,6 +145,20 @@ func Cleanup(app App) error {
 		}
 	}
 
+	volumes, err := client.ListVolumes(docker.ListVolumesOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, volume := range volumes {
+		if volume.Labels["com.docker.compose.project"] == "ddev"+strings.ToLower(app.GetName()) {
+			err := client.RemoveVolume(volume.Name)
+			if err != nil {
+				return fmt.Errorf("could not remove volume %s: %v", volume.Name, err)
+			}
+		}
+	}
```

Comment:
Pretty sure none of this should be needed anymore? It's pretty harmless, but I'm not sure what it would be cleaning?